_studio/shared/umc/codec/h265_dec/src/umc_h265_yuv.cpp

```diff
@@ -154,15 +160,15 @@ void H265DecYUVBufferPadded::allocate(const UMC::FrameData * frameData, const UM
         }
         else
         {
+            VM_ASSERT(m_frameData.GetInfo()->GetNumPlanes() == 3);
```

Comment:
Do you need this assert? You put check in a condition.

Reply:
yes. 1 - align with h264d. 2 - condition here NumPlanes **>** 2, so if not 3 - something wrong